### ISSUE TITLE
Xiphos crashes in Windows 11 when Module Manager deletes a module 

### ISSUE THREAD

**Issue by DavidHaslam** [crosswire/xiphos#1127]
First observed when attempting to update a module.

But it actually occurs even when removing a module.

I.e. That’s the first of the two steps involved in a module update.

System: Xiphos 4.2.1 installed in Windows 11 x64 PC (Dell Inspiron 16 7610).

The module does get deleted, but Xiphos crashes out immediately (afterwards?).

**Comment by DavidHaslam** [comment]
@karlkleinpaste 

Will this issue be fixed in the new release being prepared for March 2025 ?

**Comment by karlkleinpaste** [comment]
yes, due to sword 1.9.0 being included, works both within xiphos and via installmgr -u CLI.
